### PR TITLE
Support enum input type in launch form

### PR DESCRIPTION
**Pull request by Pianist038801**
Signed-off-by: Pianist038801 <steven@union.ai>

# TL;DR
Add enum type input in launch form

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1217

## Follow-up issue
_NA_

### CODE REVIEW

File: src/components/Launch/LaunchForm/SimpleInput.tsx [hunk 1]
```diff
@@ -2,28 +2,43 @@ import {
...
 } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import * as React from 'react';
 import { DatetimeInput } from './DatetimeInput';
 import { makeStringChangeHandler, makeSwitchChangeHandler } from './handlers';
 import { InputProps, InputType } from './types';
 import { UnsupportedInput } from './UnsupportedInput';
 import { getLaunchInputId } from './utils';
 
+const useStyles = makeStyles(theme => ({
```
Comment (jsonporter) [inline src/components/Launch/LaunchForm/SimpleInput.tsx:18]: NIT: not a blocker but do we need makeStyles here; couldn't we just send styles as a generic object?  (is there a specific reason for useStyle?)

eg, 
```
const formStyle = {
   minWidth: 100%; 
}
```

**Review by jsonporter — COMMENTED** [review]
Left one comment (NIT) but once addressed lgtm.

File: src/components/Launch/LaunchForm/SimpleInput.tsx (hunk 1 above)
Comment (Pianist038801) [reply src/components/Launch/LaunchForm/SimpleInput.tsx:18]: If we define a style that way, we should use style props of React component.
But I found that we currently use className prop for styling.

That's why I went with such solution, just tried to replicate how it's already implemented.

**Review by jsonporter — APPROVED** [review]
Lgtm

**Pull request merged**